### PR TITLE
Fixes #2782 - Edit dev-env-setup.md around GH client ID/secret + typos

### LINKED ISSUE #2782: Fix dev-env-setup.md around github client ID and client secret
So in trying to set up webcompat on another laptop, I discovered that this: 

screenshot 2019-01-28 13 39 53
...only works if you have owner status in the organization -- regular members following that link get a 404. I think we should either specify that or delete the suggestion (since it only applies to 4 people).

### PR DESCRIPTION
**Pull request by laghee**
This PR fixes issue #2782 

## Proposed PR background

* Deletes sentence that claims webcompat org members can retrieve the GH client ID & secret from settings (which only org owners can access)
* Fixes some typos and formatting inconsistencies (some of these might be subjective)

---

- [✅] I have read the [Pull Request + Code Style Guidelines](https://github.com/webcompat/webcompat.com/blob/master/docs/pr-coding-guidelines.md) thoroughly.

### CODE REVIEW

File: docs/dev-env-setup.md [hunk 1]
```diff
@@ -180,32 +182,31 @@ copy config/secrets.py.example config/secrets.py
...
 
-3. Add values for `GITHUB_CLIENT_ID` and `GITHUB_CLIENT_SECRET`. If you're part of the [webcompat GitHub organization](https://github.com/webcompat), you can [get the client ID and client secret from GitHub](https://github.com/organizations/webcompat/settings/applications/). Otherwise, create your own local application ([instructions here](https://github.com/settings/applications/new)) &mdash; when prompted for an "Authorization callback URL", use `http://localhost:5000/callback`, and take note of the client ID and client secret GitHub give you. 
+The [instructions for creating a personal access token](http://help.github.com/articles/creating-an-access-token-for-command-line-use) are given on GitHub. Select public_repo to grant access to the public repositories through the personal access token. Once you have created the token you can add it in the variable `OAUTH_TOKEN = ""`.
```
Comment (karlcow) [inline docs/dev-env-setup.md]: to get a consitent list formatting you need to make each subsequent [paragraphs with 4 leading spaces](https://daringfireball.net/projects/markdown/syntax#list).

```suggestion
    The [instructions for creating a personal access token](http://help.github.com/articles/creating-an-access-token-for-command-line-use) are given on GitHub. Select public_repo to grant access to the public repositories through the personal access token. Once you have created the token you can add it in the variable `OAUTH_TOKEN = ""`.
```

File: docs/dev-env-setup.md [hunk 2]
```diff
@@ -180,32 +182,31 @@ copy config/secrets.py.example config/secrets.py
...
 
-> Note: Cloud 9 users should use `http://yourapp.c9users.io:8000/callback`for the Authorization callback URL instead
+2.  Add values for `GITHUB_CLIENT_ID` and `GITHUB_CLIENT_SECRET`. Create your own local application ([instructions here](https://github.com/settings/applications/new)) &mdash; when prompted for an "Authorization callback URL", use `http://localhost:5000/callback`, and take note of the client ID and client secret GitHub gives you.
 
-  When you have the client ID and client secret, put them in the corresponding lines in `secrets.py`:
+> **Note**: Cloud 9 users should use `http://yourapp.c9users.io:8000/callback` for the Authorization callback URL instead.
```
Comment (karlcow) [inline docs/dev-env-setup.md]: Same thing here. 
```suggestion
    > **Note**: Cloud 9 users should use `http://yourapp.c9users.io:8000/callback` for the Authorization callback URL instead.
```

and so on.

Comment (laghee) [reply docs/dev-env-setup.md]: I think I got all of the indents under the numbered list. If I missed anything, LMK.

File: docs/dev-env-setup.md (hunk 1 above)
Comment (laghee) [reply docs/dev-env-setup.md]: Aha! I was trying to figure out what the orange highlighting meant in certain places in the editor... and now I know ❤️ . Changed 👍 .

**Comment by laghee** [comment]
> different markdown style ;)

Oh, weird, you're right -- I didn't even notice that! Apparently my vscode markdown plugin is very aggressive about changing even without any explicit approval... how annoying. I think I can roll those back by mucking with settings if necessary.

File: docs/dev-env-setup.md [hunk 3]
```diff
@@ -67,11 +73,11 @@ npm run setup
...
 **Note**: if you got an error message, you may need to [install pip](#installing-pip) before running `npm run setup` again.
 
-**Note**: if you get an error message about missing rights to install the setup through npm, please *do not run `sudo npm`*. You just need to [fix you permissions](https://coderwall.com/p/t2mc9g/don-t-sudo-npm) for `usr/local`.
+**Note**: if you get an error message about missing rights to install the setup through npm, please _do not run `sudo npm`_. You just need to [fix your permissions](https://coderwall.com/p/t2mc9g/don-t-sudo-npm) for `usr/local`.
 
-### Detailed setup (All platforms)
+### Detailed setup (all platforms)
 
-#### Installing Windows Substem for Linux
+#### Installing Windows Subsystem for Linux
```
Comment (miketaylr) [inline docs/dev-env-setup.md]: ha, nice find.

**Comment by miketaylr** [comment]
> I think I can roll those back by mucking with settings if necessary.

I'm not sure it really matters, as long as the end result is the same.

**Pull request merged**